pkg/updatestrategy/aws_asg.go

```diff
@@ -25,7 +25,8 @@ const (
 	clusterIDTagPrefix               = "kubernetes.io/cluster/"
 	resourceLifecycleOwned           = "owned"
 	kubeAutoScalerEnabledTagKey      = "k8s.io/cluster-autoscaler/enabled"
-	nodePoolTag                      = "NodePool"
+	nodePoolTagLegacy                = "NodePool"
```

Comment:
Do we even use it?

Reply:
This is still the tag used in `aws_asg.go` to find the right asg for a node pool. I would like to change this but will do so separately.